File: app/src/main/java/com/BlizzardArmory/warcraft/Talents/SpecializationData/GenderDescription.java
package com.BlizzardArmory.warcraft.Talents.SpecializationData;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class GenderDescription {

    @SerializedName("male")
    @Expose
    private String male;
    @SerializedName("female")
    @Expose
    private String female;

    /**
     * No args constructor for use in serialization
     */
    public GenderDescription() {
    }

    /**
     * @param female
     * @param male
     */
    public GenderDescription(String male, String female) {
        super();
        this.male = male;
        this.female = female;
    }

    public String getMale() {
        return male;
    }

    public void setMale(String male) {
        this.male = male;
    }

    public String getFemale() {
        return female;
    }

    public void setFemale(String female) {
        this.female = female;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/talents/specializationdata/GenderDescription.kt
package com.BlizzardArmory.model.warcraft.talents.specializationdata

import com.google.gson.annotations.SerializedName


/**
 * The type Gender description.
 */
data class GenderDescription(

        @SerializedName("male")
        var male: String,

        @SerializedName("female")
        var female: String

)